app/src/main/java/org/gdg/frisbee/android/fragment/FirstStartStep1Fragment.java

```diff
@@ -50,6 +51,7 @@
 
 public class FirstStartStep1Fragment extends Fragment {
 
+    private static final String ARG_SELECTED_CHAPTER = "selected_chapter";

```

Comment:
I thought about that and decided to define it twice because they are not actually related. They are for saving the state of the Activity and Fragment. And usually the constants for Intent extras and saving the instance state are defined in the respected files.